test/support/vcr.rb

```diff
@@ -2,6 +2,8 @@
 require "vcr"
 require "active_support/concern"
 
+WebMock.allow_net_connect!
```

Comment:
We're adding a similar line here in #5487. I think it would be easier to follow what is happening if we amended #5487 to reflect what we want.